package/src/components/Poll/components/PollInputDialog.tsx

```diff
@@ -0,0 +1,117 @@
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+import { useTheme, useTranslationContext } from '../../../contexts';
+
+export type PollInputDialogProps = {
+  closeDialog: () => void;
+  onSubmit: (text: string) => void;
+  title: string;
+  visible: boolean;
+  initialValue?: string;
+};
+
+export const PollInputDialog = ({
+  closeDialog,
+  initialValue = '',
+  onSubmit,
+  title,
+  visible,
+}: PollInputDialogProps) => {
+  const { t } = useTranslationContext();
+  const [dialogInput, setDialogInput] = useState(initialValue);
+
+  const {
+    theme: {
+      colors: { accent_dark_blue, black, white },
+      poll: {
+        inputDialog: {
+          button,
+          buttonContainer,
+          container,
+          input,
+          title: titleStyle,
+          transparentContainer,
+        },
+      },
+    },
+  } = useTheme();
+
+  return (
+    <Modal animationType='fade' onRequestClose={closeDialog} transparent={true} visible={visible}>
+      <KeyboardAvoidingView
+        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+        style={[styles.transparentContainer, transparentContainer]}
+      >
+        <View style={[styles.container, { backgroundColor: white }, container]}>
+          <Text style={[styles.title, { color: black }, titleStyle]}>{title}</Text>
+          <TextInput
+            autoFocus={true}
+            onChangeText={setDialogInput}
+            placeholder={t<string>('Ask a question')}
+            style={[styles.input, { color: black }, input]}
+            value={dialogInput}
+          />
+          <View style={[styles.buttonContainer, buttonContainer]}>
+            <TouchableOpacity onPress={closeDialog}>
+              <Text style={[styles.button, { color: accent_dark_blue }, button]}>
+                {t<string>('Cancel')}
+              </Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              onPress={() => {
+                onSubmit(dialogInput);
+                closeDialog();
+              }}
+              style={{ marginLeft: 32 }}
+            >
+              <Text style={[styles.button, { color: accent_dark_blue }, button]}>
+                {t<string>('SEND')}
+              </Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+      </KeyboardAvoidingView>
+    </Modal>
+  );
+};
+
+const styles = StyleSheet.create({
+  button: { fontSize: 17, fontWeight: '500' },
+  buttonContainer: { flexDirection: 'row', justifyContent: 'flex-end', marginTop: 52 },
+  container: {
+    backgroundColor: 'white',
+    borderRadius: 16,
+    paddingBottom: 20,
+    paddingHorizontal: 16,
+    paddingTop: 32,
+    width: '80%',
+  },
+  input: {
+    alignItems: 'center',
+    borderColor: 'gray',
+    borderRadius: 18,
+    borderWidth: 1,
+    fontSize: 16,
+    height: 36,
+    marginTop: 16,
+    padding: 0,
+    paddingHorizontal: 16,
+  },
+  title: { fontSize: 17, fontWeight: '500', lineHeight: 20 },
```

Comment:
Yeah, was a bit odd to me too since it's like this in the designs. I could move it to `16` but this is anyway something they can override through the theme so it doesn't really matter. If you prefer I don't mind lowering it, though.

Reply:
Yeah, just take it as a nitpeak 😛

Reply:
I'll just do the change, you're right that it doesn't make sense that it's different